<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>XbeeAddDialog</class>
 <widget class="QDialog" name="XbeeAddDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>268</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>XBee 장치 추가</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_model">
     <property name="text">
      <string>장치 모델</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_model"/>
   </item>
   <item>
    <widget class="QLabel" name="label_parent">
     <property name="text">
      <string>부모 장치</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_parent"/>
   </item>
   <item>
    <widget class="QLabel" name="label_port">
     <property name="text">
      <string>포트</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_port"/>
   </item>
   <item>
    <widget class="QLabel" name="label_baudrate">
     <property name="text">
      <string>통신 속도</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="comboBox_baudrate_add">
     <property name="editable">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>9600</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>115200</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_ch">
     <property name="text">
      <string>채널 (CH)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_ch"/>
   </item>
   <item>
    <widget class="QLabel" name="label_panid">
     <property name="text">
      <string>PAN ID</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_panid"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_refresh">
       <property name="text">
        <string>새로고침</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_connect">
       <property name="text">
        <string>XBee 연결</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_disconnect">
       <property name="text">
        <string>XBee 해제</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label_sn">
     <property name="text">
      <string>시리얼 번호</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_sn">
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_name">
     <property name="text">
      <string>장치 이름</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_name"/>
   </item>
   <item>
    <widget class="QLabel" name="label_location">
     <property name="text">
      <string>설치 위치</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_location"/>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit_log"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_buttons">
     <item>
      <widget class="QPushButton" name="button_add">
       <property name="text">
        <string>장치 추가</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_cancel">
       <property name="text">
        <string>취소</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
